src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationemployment/api/integration/resource/SARReadinessProfileGetShould.kt

```diff
@@ -66,17 +69,86 @@ class SARReadinessProfileGetShould : SARReadinessProfileTestCase() {
     assertAddReadinessProfileIsOk(prisonNumber, profileRequestOfKnownPRN)
     val expectedProfile = profileJsonOfKnownPRN
 
-    val sarResult = assertGetSARResponseIsOk(prn = prisonNumber)
+    val sarResult = assertGetSARResponseIsOk(expectedProfileAsJson = expectedProfile, prn = prisonNumber)
```

Comment:
Instead of `prn` we can use `prisonNumber`

Reply:
I used `prn` at some points, as this term is coined from SAR API specification (along with crn)
I aware SAR team was informed the better name _prison number_, but seems they have not yet adopt it.
Until the renaming happen, I prefer using SAR terms for consistency in SAR domain

Reply:
I removed most PRN and CRN but still keep these at request params, as per SAR API spec.